### ISSUE TITLE
Manage Large data | Candle Chart Stucks | Scrolling to see all data

### ISSUE THREAD

**Issue by zahidch052** [syncfusion/flutter-widgets#1161]
Hi, I'm working on a financial app and using Candle Stick Chart, I'm getting data from sever in thousands, Chart take a while to load data, that's fine, but even after loading, I try to zoom it, the chart stucks and app is stuck until that zoom interaction is processed, also I want to add scroll behavior in order to see all the data loaded on chart.

So I want help with two major things:
1- How to handle extensive data?

2- How to add scrolling to view all the data on the chart.

Note: zoom and pane is required as well.

Thanks

**Comment by zahidch052** [comment]
I tried a one of your examples in some other user issues, and scrolling works fine on DateTime(YYYY,MM,DD), but from server data is coming in the form of epochtime  , scrolling after conversion to DateTime it works fine on X-axis but scroll don't work.

please check this video: [https://vimeo.com/816226979 ](https://vimeo.com/816226979)

**Comment by LokeshPalani** [comment]
Hi @zahidch052,

We have checked the provided video and it is not opening. We attempted to replicate the issue on our end but were unsuccessful. We are uncertain of the exact scenario in which the issue is occurring. Therefore, we kindly request you attempt to replicate the issue using the test sample attached below and inform us of the results. This will enable us to better assist you.

Regards,
Lokesh.
[chart_469461.zip](https://github.com/syncfusion/flutter-widgets/files/11661308/chart_469461.zip)
